lib/static/new-ui/types/store.ts

```diff
@@ -59,35 +77,54 @@ export interface SuiteState {
 
 export interface BrowserState {
     shouldBeShown: boolean;
+    retryIndex: number;
+    // True if test is not shown because of its status. Useful when computing counts by status.
+    isHiddenBecauseOfStatus: boolean;
```

Comment:
Since you are only using "true" or not defining the property at all, maybe we should describe it as `isHiddenBecauseOfStatus?: true`?

Reply:
Made this field optional. But I see no reason for using `true` instead of `boolean`. Right now it's either true or absent, but why prohibit setting it to false.